maven/maven.bzl

```diff
@@ -159,11 +219,79 @@ def _should_include_dependency(dep):
         not dep.optional
     )
 
+def _cut(str, *to_phrases):
+    start_idx = 0
+    for phrase in to_phrases:
+        idx = str.find(phrase, start_idx)
+        if idx == -1:
+            return ("", False)
+        start_idx = idx + len(phrase)
+
+    return (str[start_idx:].strip(), True)
+
+def _extract_android_package(ctx, manifest):
+    m = ctx.read(manifest)
+    pkg = ""
+    for tag in m.split("<"):
+        if tag.startswith("manifest"):
+            # matching package\s*=\s*"|'pkg"|'
+            quot_pkg_quot_trailing, ok = _cut(tag, "package", "=")
+            if not ok:
+                fail("can't find package = in |%s| of |%s|" % (tag, m))
+            start_quote = quot_pkg_quot_trailing[0]
+            if start_quote not in ("'", "\""):
+                fail("can't find quote = in |%s| of |%s|" % (quot_pkg_quot_trailing, m))
+            end_idx = quot_pkg_quot_trailing.find(start_quote, 1)
+            pkg = quot_pkg_quot_trailing[1:end_idx]
+            break
+
+    if pkg == "":
```

Comment:
empty strings? Shouldn't we be using None and `if pkg:`. (nit: this works, but seems odd)